### PR TITLE
Update of v6.3 release notes (Part 3)

### PR DESCRIPTION
**Pull request by sf1919**
**Description of work.**
To reduce merge conflicts release notes added during release have been added as separate files. This PR collates all these notes into the main V6.3 release notes and undertakes cleaning and formatting to improve quality and consistency.

In this PR the following release notes have been updated:-

- Direct Geometry
- Framework
- Indirect Geometry
- Workbench

**To test:**

- remove `build/docs/doctrees`
- build the documents again and check there are no warnings
- Check that the links work
- Check images display as expected
- Check the information is still correct
- Grammar/Spelling check

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v6.3.0/framework.rst [hunk 1]
```diff
@@ -36,6 +36,7 @@ Bugfixes
 ########
 - Fixed a bug with :ref:`CalculatePlaczek <algm-CalculatePlaczek>` algorithm for computing Placzek correction factors that fixed the previously implemented formula for transforming k to e, and the summation for second order Placezek corrections.
 - :ref:`ConvertAxesToRealSpace <algm-ConvertAxesToRealSpace>` no longer crashes Mantid if using an invalid file.
+- Calculation of log binning factor in :ref:`DiffractionFocussing <algm-DiffractionFocussing>` adjusted to remove a small discrepancy between total x range before\after rebinning.
```
Comment (ConorMFinn) [inline docs/source/release/v6.3.0/framework.rst:39]: I think this backslash here needs escaping, as it's not appearing in the built docs. Or just a forward slash
```suggestion
- Calculation of log binning factor in :ref:`DiffractionFocussing <algm-DiffractionFocussing>` adjusted to remove a small discrepancy between total x range before\\after rebinning.
```
```suggestion
- Calculation of log binning factor in :ref:`DiffractionFocussing <algm-DiffractionFocussing>` adjusted to remove a small discrepancy between total x range before/after rebinning.
```

File: docs/source/release/v6.3.0/mantidworkbench.rst [hunk 2]
```diff
@@ -89,5 +90,6 @@ Bugfixes
 - Transposing data (i.e. swapping x and y axes) of ``2D MD`` workspace, now works without error.
 - Fixed issues with the colorbar autoscale not updating correctly on zoom.
 - Stopped the ROI rectangle selection extents jumping discontinuously when the user tries to resize beyond the extent of the colorfill axes towards the line plot axes.
+- :ref:`SliceViewer` will no longer dynamically rebin when viewing an ``MDHisto`` workspace that has been modified by a binary operation (e.g. MinusMD).
```
Comment (ConorMFinn) [inline docs/source/release/v6.3.0/mantidworkbench.rst:93]: Should the MinusMD example link to the algorithm?

**Review by cailafinn — CHANGES_REQUESTED** [review]
Builds properly and looks good, just a couple of nitpicks

**Review by cailafinn — APPROVED** [review]
Looks good to me now, nice one 👍

**Pull request merged**